subsys/suit/cache/Kconfig

```diff
@@ -52,6 +52,13 @@ config SUIT_CACHE0_ERASE_ON_ENVELOPE_STORED
 	depends on SUIT_CACHE_RW
 	default y
 
+config SUIT_CACHEX_ERASE_ON_INIT
+	bool "Always erase the DFU cache partition 1 .. n on initialization"
+	default y
```

Comment:
This is changing the behavior, I think it should be `default n`

Reply:
Removed the default